app/src/main/res/layout-sw600dp-land/topic_lessons_story_summary.xml

```diff
@@ -141,37 +141,37 @@
                 android:layout_height="wrap_content"
                 android:layout_gravity="center_horizontal|bottom"
                 android:contentDescription="@{isListExpanded ? @string/hide_chapter_list : @string/show_chapter_list}"
-                app:srcCompat="@drawable/ic_arrow_drop_down_black_24dp"
                 app:isRotationAnimationClockwise="@{isListExpanded}"
-                app:rotationAnimationAngle="@{180f}" />
+                app:rotationAnimationAngle="@{180f}"
+                app:srcCompat="@drawable/ic_arrow_drop_down_black_24dp" />
             </FrameLayout>
           </LinearLayout>
         </LinearLayout>
-
-        <LinearLayout
+        <!-- FrameLayout is used here instead of LinearLayout to properly adjust the spacing of the
+            dashed divider by allowing the divider to render slightly on top of the container. -->
+        <FrameLayout
           android:id="@+id/chapter_list_container"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
           android:background="@color/color_def_white_light"
           android:orientation="vertical"
           android:visibility="@{isListExpanded? View.VISIBLE : View.GONE}">
 
-          <View
-            android:id="@+id/topic_play_story_dashed_line_view"
-            android:layout_width="match_parent"
-            android:layout_height="2dp"
-            android:layout_gravity="center_horizontal"
-            android:background="@drawable/dashed_divider" />
-
           <androidx.recyclerview.widget.RecyclerView
             android:id="@+id/chapter_recycler_view"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginTop="8dp"
-            android:layout_marginBottom="8dp"
+            android:layout_marginTop="1.5dp"
```

Comment:
Margin should never be 1.5dp. Ideally it should be a multiple of 4. But if needed you can write it as 2dp.

Reply:
done